python/paddle/fluid/layers/nn.py

```diff
@@ -12075,7 +12075,10 @@ def logical_and(x, y, out=None, name=None):
             res = paddle.logical_and(x, y)
             print(res.numpy()) # [True False False False]
     """
-
+    if x.shape != y.shape:
+        raise TypeError(
+            'Input tensors must be same shape, but receive x \'s shape: %s, y \'s shape: %s '
```

Comment:
Tiny English grammar issue:

"but receive" -> "but received"

The Variable's **was received in the past**, not a common thing.

I found most of PaddlePaddle codes used "but received", few of them used "but receive", please correct it.

Reply:
thx